gradle/gradle-mvn-push.gradle

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-apply plugin: 'maven'
+apply plugin: 'maven-publish'
```

Comment:
We can just delete this file to be honest; I intend to replace it entirely with a gradle plugin.

Reply:
Deleted!